tests/core/client.spec.ts

```diff
@@ -1,61 +1,99 @@
-};
-
-const INVOICE_ID = 'TRnwXeAkuLQihe22mJs7J4';
-
-const loginAndGetPairingCode = async (): Promise<any> => {
+let STORE_ID = '';
+const WINDOW_WIDTH = 1920;
+const WINDOW_HEIGHT = 1080;
+
+let INVOICE_ID = '';
+const HEADLESS = true;
+
+const loginAndGetPairingCode = async (): Promise<{
+  browser: Browser;
+  page: Page;
+  pairingCode: string;
+}> => {
   const newTokenName = 'autotest ' + new Date().getTime();
 
-  const browser = await puppeteer.launch({ headless: true }).then(
-    v => v, // if success, passthrough
-    // if error, check for env and ignore sandbox and warn.
-    err => {
-      if (IGNORE_SANDBOX_ERROR === '1') {
-        console.warn(
-          'WARNING!!! Error occurred, Chromium will be started ' +
-            "without sandbox. This won't guarantee success.",
-        );
-        return puppeteer.launch({
-          headless: true,
-          ignoreDefaultArgs: ['--disable-extensions'],
-          args: ['--no-sandbox', '--disable-setuid-sandbox'],
-        });
-      } else {
-        console.warn(
-          'If "No usable sandbox!" error, retry test with ' +
-            'BTCPAY_IGNORE_SANDBOX_ERROR=1',
-        );
-        throw err;
-      }
-    },
-  );
+  const browser = await puppeteer
+    .launch({
+      headless: HEADLESS,
+      args: ['--window-size=' + WINDOW_WIDTH + ',' + WINDOW_HEIGHT],
+    })
+    .then(
+      v => v, // if success, passthrough
+      // if error, check for env and ignore sandbox and warn.
+      err => {
+        if (IGNORE_SANDBOX_ERROR === '1') {
+          console.warn(
+            'WARNING!!! Error occurred, Chromium will be started ' +
+              "without sandbox. This won't guarantee success.",
+          );
+          return puppeteer.launch({
+            headless: HEADLESS,
+            ignoreDefaultArgs: ['--disable-extensions'],
+            args: [
+              '--no-sandbox',
+              '--disable-setuid-sandbox',
+              '--window-size=' + WINDOW_WIDTH + ',' + WINDOW_HEIGHT,
+            ],
+          });
+        } else {
+          console.warn(
+            'If "No usable sandbox!" error, retry test with ' +
+              'BTCPAY_IGNORE_SANDBOX_ERROR=1',
+          );
+          throw err;
+        }
+      },
+    );
   const page = (await browser.pages())[0];
-  await page.goto('https://testnet.demo.btcpayserver.org/Account/Login');
+  await page.setViewport({ width: WINDOW_WIDTH, height: WINDOW_HEIGHT });
+  try {
+    await page.goto(URL + '/Account/Login');
+  } catch (e) {
+    if (e.message === `net::ERR_CONNECTION_REFUSED at ${URL}/Account/Login`) {
+      browser.close();
+      console.log(
+        'Please start docker container locally:\n' +
+          'docker run -p 127.0.0.1:49392:49392 junderw/btcpay-client-test-server',
+      );
+      return {
+        page,
+        browser,
+        pairingCode: '',
+      };
+    }
+    throw e;
+  }
 
   await page.type('#Email', USER_NAME);
   await page.type('#Password', PASSWORD);
```

Comment:
The login screen is already pre-populated with these login credentials so entering the username and password again will append to those values and lead to a failed login. See attached screenshot.
![loggedin](https://user-images.githubusercontent.com/41004/106614586-6f652780-6563-11eb-9911-194822eb8a55.png)

Reply:
Done